test/ExpressiveInstallerTest/OptionalPackagesTestCase.php

```diff
@@ -47,12 +50,9 @@
     /**
      * Assert that the installer contains a specification for the package.
      *
-     * @param string $package
-     * @param OptionalPackages $installer
-     * @param null|string $message
-     * @throws \PHPUnit_Framework_AssertionFailedError
+     * @throws AssertionFailedError
      */
-    public static function assertPackage($package, OptionalPackages $installer, $message = null)
+    public static function assertPackage(string $package, OptionalPackages $installer, ?string $message = null)
```

Comment:
As said before, the type should be just `string` as variable has default `null` value